scripts/upload_files_to_bucket.sh

```diff
@@ -0,0 +1,54 @@
+#!/bin/bash -e
```

Comment:
Any reason to not use python? Then we could have flags instead of positional args.

Reply:
This came as part of #628, which included a python wrapper around this shell script. The cleanup script is still forthcoming so I expect it will be the same. 

I guess we could cut out the middle-man and just call the bash part in a subprocess directly.

Reply:
I'll make a note in #599.